pkg/arguments/arguments.go

```diff
@@ -333,11 +334,13 @@ func MarkRegionDeprecated(parentCmd *cobra.Command, childrenCmds []*cobra.Comman
 		})
 		currentRun := cmd.Run
 		cmd.Run = func(c *cobra.Command, args []string) {
-			outputFlag := cmd.Flag("output")
-			regionFlag := cmd.Flag("region")
-			hasChangedOutputFlag := outputFlag != nil && outputFlag.Value.String() != outputFlag.DefValue
-			hasChangedRegionFlag := regionFlag != nil && regionFlag.Value.String() != regionFlag.DefValue
-			if hasChangedRegionFlag && !hasChangedOutputFlag {
+			hasRegion := false
+			cmd.Parent().PersistentFlags().VisitAll(func(flag *pflag.Flag) {
+				if flag.Name == regionFlagName {
+					hasRegion = true
+				}
+			})
+			if cmd.Flags().Lookup(outputFlagName).Value.String() == "" && hasRegion {
```

Comment:
note on the value comparison choice - `.Changed` does not seem to function how I would expect with the output flag, it's always true even when not used. This comparison seems to work fine in it's replacement